test/e2e/bpmn.rendering.test.ts

```diff
@@ -129,3 +130,19 @@ describe('no visual regression', () => {
     expect(image).toMatchImageSnapshot(imageSnapshotConfigurator.getConfig(fileName));
   });
 });
+
+it.each([
+  [FitType[FitType.Default], FitType.Default],
+  [FitType[FitType.Horizontal], FitType.Horizontal],
+  [FitType[FitType.Vertical], FitType.Vertical],
+  [FitType[FitType.HorizontalVertical], FitType.HorizontalVertical],
+])(`load options: fit %s`, async (title: string, fitType: FitType) => {
+  const bpmnDiagramPreparation = new BpmnDiagramPreparation(new Map<string, BpmnLoadMethod>([]), { name: 'non-regression' }, 'non-regression', { fitType });
```

Comment:
:warning: this should be moved in a dedicated test files or with navigation tests. Rendering tests are focussed on the way the BPMN elements are rendered. Here we want to test the way the bpmn diagram fits the viewport which is something different

Reply:
For me,  **Fit** is an option of rendering. It's why I put here.
Maybe we need to rename this test file, if we want to separate them.

Reply:
Please extract the new tests to a new file called diagram rendering or something similar
Putting everything together is messy and introduce too much changes

Reply:
I'll do it 😄